_posts/2017-10-30-user-secrets.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: Safe storage of app secrets during development in ASP.NET Core
+time: 2017-10-30
+---
+
+The title of this article is a direct copy of [this](https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?tabs=visual-studio) great article on Microsoft's website. The reason I am writing this post is because one should know the options when developping and running [ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/) application locally and does not want to commit sensitive data like password, API key, etc... So I am hoping to contribute to spreading the words with that post :)
```

Comment:
_developing_ 

The second sentence is a little clunky; perhaps something like "..one should know the options that permit you to develop locally without risking committing sensitive data like passwords, API keys etc" 

_this_ post (you got it right in the second sentence!